@mfasDa : This seems to be a genuine error:
```
/sw/SOURCES/O2/6864-slc8_x86-64/0/Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx:341:25: error: statement should be inside braces [readability-braces-around-statements]
```

--- a/Detectors/EMCAL/reconstruction/src/CaloRawFitter.cxx
+++ b/Detectors/EMCAL/reconstruction/src/CaloRawFitter.cxx
@@ -55,6 +55,8 @@ int CaloRawFitter::getErrorNumber(CaloRawFitter::RawFitterError_t fiterror)
       return 2;
     case RawFitterError_t::BUNCH_NOT_OK:
       return 3;
+    case RawFitterError_t::LOW_SIGNAL:
+      return 4;
   };
   // Silence compiler warnings for false positives
   // can never enter here due to usage of enum class

--- a/Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx
+++ b/Detectors/EMCAL/workflow/src/RawToCellConverterSpec.cxx
@@ -219,11 +219,10 @@ void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
         LOG(DEBUG4) << decoder.getRCUTrailer();
       }
       // Apply zero suppression only in case it was enabled
-      std::cout << "ALTRO CFG1" << std::bitset<32>(decoder.getRCUTrailer().getAltroCFGReg1()) << ", CFG2: " << std::bitset<32>(decoder.getRCUTrailer().getAltroCFGReg2()) << std::endl;
       if (decoder.getRCUTrailer().hasZeroSuppression()) {
-        std::cout << "Zero suppression enabled" << std::endl;
+        LOG(DEBUG3) << "Zero suppression enabled";
       } else {
-        std::cout << "Zero suppression disabled" << std::endl;
+        LOG(DEBUG3) << "Zero suppression disabled";
       }
       mRawFitter->setIsZeroSuppressed(decoder.getRCUTrailer().hasZeroSuppression());
 
@@ -231,6 +230,7 @@ void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
       int iSM = feeID / 2;
 
       // Loop over all the channels
+      int nBunchesNotOK = 0;
       for (auto& chan : decoder.getChannels()) {
 
         int iRow, iCol;
@@ -240,7 +240,7 @@ void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
           iCol = map.getColumn(chan.getHardwareAddress());
           chantype = map.getChannelType(chan.getHardwareAddress());
         } catch (Mapper::AddressNotFoundException& ex) {
-          std::cerr << ex.what() << std::endl;
+          LOG(ERROR) << ex.what();
           continue;
         };
 
@@ -319,18 +319,27 @@ void RawToCellConverterSpec::run(framework::ProcessingContext& ctx)
             fitResults.setTime(0.);
           }
         } catch (CaloRawFitter::RawFitterError_t& fiterror) {
-          if (mNumErrorMessages < mMaxErrorMessages) {
-            LOG(ERROR) << "Failure in raw fitting: " << CaloRawFitter::createErrorMessage(fiterror);
-            mNumErrorMessages++;
-            if (mNumErrorMessages == mMaxErrorMessages) {
-              LOG(ERROR) << "Max. amount of error messages (" << mMaxErrorMessages << " reached, further messages will be suppressed";
+          if (fiterror != CaloRawFitter::RawFitterError_t::BUNCH_NOT_OK) {
+            // Display
+            if (mNumErrorMessages < mMaxErrorMessages) {
+              LOG(ERROR) << "Failure in raw fitting: " << CaloRawFitter::createErrorMessage(fiterror);
+              mNumErrorMessages++;
+              if (mNumErrorMessages == mMaxErrorMessages) {
+                LOG(ERROR) << "Max. amount of error messages (" << mMaxErrorMessages << " reached, further messages will be suppressed";
+              }
+            } else {
+              mErrorMessagesSuppressed++;
             }
           } else {
-            mErrorMessagesSuppressed++;
+            LOG(DEBUG2) << "Failure in raw fitting: " << CaloRawFitter::createErrorMessage(fiterror);
+            nBunchesNotOK++;
           }
           mOutputDecoderErrors.emplace_back(feeID, ErrorTypeFEE::ErrorSource_t::FIT_ERROR, CaloRawFitter::getErrorNumber(fiterror));
         }
         currentCellContainer->emplace_back(CellID, fitResults.getAmp() * CONVADCGEV, fitResults.getTime(), chantype);
+      }
+      if (nBunchesNotOK) {
+        LOG(DEBUG) << "Number of failed bunches: " << nBunchesNotOK;
       }
     }
   }

